orchestrators/ProcurementOperations/src/Services/GrIrAccrualService.php

```
@@ -134,126 +106,46 @@ public function matchWithInvoice(
-
-        if ($matchedAmount->getAmount() >= $accrual->accrualAmount->getAmount()) {
-            throw new \InvalidArgumentException(
-                'Matched amount must be less than accrual amount for partial match'
-            );
-        }
+        Money $varianceAmount,
+        string $varianceReason,
+        string $matchedBy,
+    ): GrIrAccrualData {
+        $accrual = $this->getAccrual($accrualId);
+        if ($accrual === null) { throw new \InvalidArgumentException("Accrual not found: {$accrualId}"); }
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '118', column '44'). (undefined)

(MissingImport)